twitterscraper/main.py

```diff
@@ -113,13 +113,35 @@ def main():
                 with open(args.output, "w", encoding="utf-8") as output:
                     if args.csv:
                         f = csv.writer(output, delimiter=";")
-                        f.writerow(["username", "fullname","user_id", "tweet_id", "tweet_url", "timestamp","timestamp_epochs",
-                                    "replies", "retweets", "likes", "is_retweet", "retweeter_username" , "retweeter_userid" ,
-                                    "retweet_id","text", "html"])
+                        f.writerow([
+                            # user name & id
+                            "screen_name", "username", "user_id",
+                            # tweet basic data
+                            "tweet_id", "tweet_url", "timestamp", "timestamp_epochs",
+                            # tweet text
+                            "text", "text_html", "links", "hashtags",
+                            # tweet media
+                            "has_media", "img_urls", "video_url",
+                            # tweet actions numbers
+                            "likes", "retweets", "replies", "is_replied",
+                            # detail of reply to others
+                            "is_reply_to", "parent_tweet_id", "reply_to_users"
+                        ])
                         for t in tweets:
-                            f.writerow([t.username, t.fullname,t.user_id, t.tweet_id, t.tweet_url, t.timestamp, t.timestamp_epochs,
-                                        t.replies, t.retweets, t.likes, t.is_retweet, t.retweeter_username , t.retweeter_userid ,
-                                        t.retweet_id, t.text, t.html])
+                            f.writerow([
+                                # user name & id
+                                t.screen_name, t.username, t.user_id,
+                                # tweet basic data
+                                t.tweet_id, t.tweet_url, t.timestamp, t.timestamp_epochs,
+                                # tweet text
+                                t.text, t.text_html, t.links, t.hashtags,
+                                # tweet media
+                                t.has_media, t.img_urls, t.video_url,
+                                # tweet actions numbers
+                                t.likes, t.retweets, t.replies, t.is_replied,
+                                # detail of reply to others
+                                t.is_reply_to, t.parent_tweet_id, t.reply_to_users
```

Comment:
same as above :)

Reply:
I learned from your comments and really convinced.
I commented too much in my PR and reduced conservative. I should've put comments in `Tweet` class's doc-string, not in internal processes.